From e66d174f6e97c6c4ba359f968486afec9e1de950 Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Tue, 27 Jun 2017 09:41:13 +0800
Subject: [PATCH] arm: configs: rockchip: enable SND_ROCKCHIP_HDMI_ANALOG

This patch enable SND_ROCKCHIP_HDMI_ANALOG for built-in hdmi sound
and external codec which share the same i2s controller.

Change-Id: I3ac29ba12f75c938e015d2b77a3b749888c2e4c2
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 911c3423bf74..5c1ac075e0e9 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -435,6 +435,7 @@ CONFIG_SND_USB_AUDIO=y
 CONFIG_SND_SOC=y
 CONFIG_SND_SOC_ROCKCHIP=y
 CONFIG_SND_SOC_ROCKCHIP_SPDIF=y
+CONFIG_SND_SOC_ROCKCHIP_HDMI_ANALOG=y
 CONFIG_SND_SOC_ROCKCHIP_HDMI_DP=y
 CONFIG_SND_SOC_ROCKCHIP_CDNDP=y
 CONFIG_SND_SOC_BT_SCO=y
-- 
2.35.3

